<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualisationWidget</class>
 <widget class="QMainWindow" name="VisualisationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save Figure</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save Figure As</string>
   </property>
  </action>
  <action name="actionSelect_Variables">
   <property name="text">
    <string>Select Variables</string>
   </property>
  </action>
  <action name="actionSelect_Variables_2">
   <property name="text">
    <string>Select Variables</string>
   </property>
  </action>
  <action name="actionSelect_Vars">
   <property name="text">
    <string>Select Variables</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
